src/create-db/download-db-dependencies.ps1

```diff
@@ -20,11 +20,13 @@ foreach ($dependency in $dependencies) {
     $version = $dependency.version
     $url = $dependency.nugetUrl
     $nugetOutput = "$dependencyOutputFolder/$packageName.nupkg"
+
+    $headers = If ($dependency.authToken) { @{ "Authorization" = "Bearer $($dependency.authToken)" } } Else { @{} };
     Write-Host "Downloading $packageName.$version"
     Remove-Item $nugetOutput -Force -Recurse -ErrorAction Ignore
 
     try {
-        Invoke-WebRequest $url -OutFile $nugetOutput
+        Invoke-WebRequest $url -OutFile $nugetOutput -Headers $headers
```

Comment:
Will this complain if you have a -Headers and an empty value?

Reply:
I tried it out and it seemed to work just fine. No errors or complaints 🙂